core/storage/user/gae_models.py

```diff
@@ -97,6 +97,61 @@ def get_by_normalized_username(cls, normalized_username):
             cls.normalized_username == normalized_username).get()
 
 
+class ActivitiesCompletedByLearnerModel(base_models.BaseModel):
+    """Keeps track of all the explorations and collections completed by the
+    learner.
+
+    Instances of this class are keyed by the user id.
+    """
+    # IDs of all the explorations completed by the user.
+    completed_exp_ids = ndb.StringProperty(repeated=True, indexed=True)
```

Comment:
If we have a field called completed_collection_ids then perhaps we should call this completed_exploration_ids for consistency, ditto elsewhere.